tests/integration/test_fim/test_registry/test_registry_ignore/test_ignore_registry.py

```diff
@@ -5,10 +5,10 @@
 import os
 
 import pytest
-from wazuh_testing import global_parameters
-from wazuh_testing.fim import LOG_FILE_PATH, callback_ignore, create_registry, registry_parser, \
-    KEY_WOW64_32KEY, KEY_WOW64_64KEY, generate_params, callback_detect_event, check_time_travel, \
-    modify_registry_value, REG_SZ
+from wazuh_testing import global_parameters, fim
+from wazuh_testing.fim import LOG_FILE_PATH, create_registry, registry_parser, \
```

Comment:
`fim` is being imported twice, if it is possible, we should decide one.

Reply:
Done 25b6b8e9a294aec722b3c8b4f042b11f05f9cc56